integration-tests/sanity-functional-protractor/specs/api-spec.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+//npm test -- --specs specs/api-spec.js
+
+const PublishPage = require('../page_objects/abstract-po.js');
+const AetsApi = require('../page_objects/api-po.js');
+const testDataJson = require('../data/test_data/api-data.json');
+const chakram = require('chakram');
+const chakramExpect = chakram.expect;
+const jasmineExpect = expect;
+
+describe('AET reports web application - Rest API tests', () => {
+
+    const HTTP_200_OK = 200;
+    const HTTP_404_NOT_FOUND = 404;
+    const HTTP_400_BAD_REQUEST = 400;
+
+    describe('Get metadata by suite name api - validation', () => {
+
+        it('JSON Schema validation', () => {
+            const parametersSuiteName = testDataJson.get_metadata_by_suite_name.parameters;
+            const jsonSchemaPath = require(testDataJson.get_metadata_by_suite_name.json_schema_path);
```

Comment:
Wouldn't be more clean to declare all of the requires at the beginning of the file?

Reply:
cleaned up